### PR TITLE
Changes needed for local development

### PR DESCRIPTION
**Pull request by paterczm**
${subject}

### CODE REVIEW

**Comment by jewzaam** [comment]
We have not cut a 1.2.0 version of lightblue-applications.  Why do we need to bump the minor version?

**Comment by paterczm** [comment]
Are applications versioned independently from lightblue? I assumed everything which is being worked on right now is 1.3.0-SNAPSHOT.

**Comment by jewzaam** [comment]
@paterczm yes, they're versioned separately.

**Comment by dcrissman** [comment]
It appears that the versioning bump requirement has been dropped. Updating subject.

File: common/src/main/webapp/js/lightblue-config.js [hunk 1]
```diff
@@ -1 +1 @@
-var metadataServicePath = "/app/metadata/rest-request/";
\ No newline at end of file
+var metadataServicePath = "rest-request/";
```
Comment (dcrissman) [inline common/src/main/webapp/js/lightblue-config.js]: Is this a breaking change?

**Comment by dcrissman** [comment]
@paterczm you will need to merge in changes from master

**Pull request merged**